addons/hr_recruitment_event/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Recruitment - Event',
+    'category': 'Human Resources/Recruitment',
+    'version': '1.0',
+    'depends': ['event', 'hr_recruitment'],
```

Comment:
There is no actual link between event and hr_recruitment in this module, just a mere comparison between email addresses. 

I mean, this whole feature is already achievable with real fields and linked reporting using UTMs right ?

Reply:
I talked with Geoffrey and he wants to have an other thing than just UTM link